### PR TITLE
Resolves replace `String` with `&'a str`

### PR DESCRIPTION
**Pull request by IamSaquib**
#### ✔ Checklist:

- [x] This PR has been added to [CHANGELOG.md](https://github.com/dotenv-linter/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);
- [x] Tests for the changes have been added (for bug fixes / features);
- [x] Docs have been added / updated (for bug fixes / features).

### CODE REVIEW

**Comment by IamSaquib** [comment]
#181

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add GitHub Workflow for AUR publishing [#161](https://github.com/dotenv-linter/dotenv-linter/pull/161) ([@mstruebing](https://github.com/mstruebing))
 
 ### 🔧 Changed
+- Replace `String` with `&'a str` for `DuplicatedKeyChecker` [#181](https://github.com/dotenv-linter/dotenv-linter/pull/181) ([@iamsaquib](https://github.com/iamsaquib))
```
Comment (mgrachev) [inline CHANGELOG.md:16]:
```suggestion
- Replace `String` with `&'a str` for `DuplicatedKeyChecker` [#192](https://github.com/dotenv-linter/dotenv-linter/pull/192) ([@iamsaquib](https://github.com/iamsaquib))
```

**Comment by IamSaquib** [comment]
Sure

**Comment by mgrachev** [comment]
@IamSaquib Thank you! ❤️

**Pull request merged**